### ISSUE TITLE
Subtle: Unique elements

### ISSUE THREAD

**Issue by blerude** [CortneyFitzgerald/TeaCozy#1]
https://github.com/CortneyFitzgerald/TeaCozy/blob/master/index.html#L27

Unique elements can be given ids instead of classes to show that they are one-of-a-kind.

**Comment by CortneyFitzgerald** [comment]
ok, and as for the flex stuff that wasn't exactly where it was supposed to be, is that fixed by being more specific with classes and ids instead of chains? I know I was getting stumped a little on that stuff not moving where I thought it should according to diff rules

**Comment by blerude** [comment]
Yes definitely. The further you get down a chain, the less control you have over the element. If something is in the wrong place, I usually play with flex rules and stylings starting at that element and moving up through its parents until I can isolate where the problem is taking place.

**Comment by CortneyFitzgerald** [comment]
Ohhhh, ok. Thank you for the feeeback!

Sincerely, 
        Cortney Fitzgerald
        Virtual Assistant to Caleb Linden

Sent from my iPhone

On Dec 13, 2017, at 2:34 AM, blerude <notifications@github.com> wrote:

Yes definitely. The further you get down a chain, the less control you have over the elements. If something is in the wrong place, I usually play with flex rules and stylings starting at that element and moving up through it's parents until I can isolate where the problem is taking place.
